auto_round/inference/backend.py

```diff
@@ -236,7 +247,21 @@ def mxfp4_scheme_checker(
     dtype=["float32", "float16", "bfloat16"],
     bits=[4],
     priority=0,
-    checkers=[mxfp4_scheme_checker],
+    checkers=[mxfp4_scheme_checker, feature_multiply_checker_32],
+    alias=["auto_round", "torch"],
+    requirements=["auto-round>0.7.0"],
+)
+
+# NVFP4
+
+BackendInfos["auto_round:torch_nvfp4"] = BackendInfo(
+    device=["xpu", "cuda", "cpu"],
+    packing_format=LLM_COMPRESSOR_FORMAT,
+    sym=[True],
+    dtype=["float32", "float16", "bfloat16"],
```

Comment:
please add group_size for mxfp/nvfp to make some checkers more robust

Reply:
added.